ibtk/src/lagrangian/LInitStrategy.cpp

```diff
@@ -139,6 +139,14 @@ LInitStrategy::tagCellsForInitialRefinement(const Pointer<PatchHierarchy<NDIM> >
     return;
 } // tagCellsForInitialRefinement
 
+void
+LInitStrategy::init()
+{
+    TBOX_WARNING("LInitStrategy::init()\n"
```

Comment:
I think there is no need to warn here --- just leave it empty, and note in the comments in the header that a default empty implementation is provided.